File: app/src/main/java/com/uwetrottmann/seriesguide/extensionexample/app/ExampleExtensionReceiver.java
package com.uwetrottmann.seriesguide.extensionexample.app;

import com.battlelancer.seriesguide.api.SeriesGuideExtension;
import com.battlelancer.seriesguide.api.SeriesGuideExtensionReceiver;


public class ExampleExtensionReceiver extends SeriesGuideExtensionReceiver {
    @Override
    protected int getJobId() {
        return 1;
    }

    @Override
    protected Class<? extends SeriesGuideExtension> getExtensionClass() {
        return ExampleExtensionService.class;
    }
}

File: app/src/main/java/com/uwetrottmann/seriesguide/extensionexample/app/ExampleExtensionReceiver.kt
package com.uwetrottmann.seriesguide.extensionexample.app

import com.battlelancer.seriesguide.api.SeriesGuideExtension
import com.battlelancer.seriesguide.api.SeriesGuideExtensionReceiver

class ExampleExtensionReceiver : SeriesGuideExtensionReceiver() {

    override fun getJobId(): Int = 1

    override fun getExtensionClass(): Class<out SeriesGuideExtension?> =
        ExampleExtensionService::class.java

}
